README.md

```diff
@@ -40,13 +40,12 @@ Also, you can of course not use some features outlined here. That's why the whol
 
 
 ## Feature list
-* easy to understand, easy to modify (main script with less than 120 LOC)
+* easy to understand, easy to modify
 * POSIX-compatible
-* logging-friendly
-* passphrase saved in external file, which you may protect in a better way
-* pruning included
-* privilege-separation (login scripts can have higher privilege than backup process)
-* also a good idea: dump your database before backing up
+* pretty logs
+* passphrase protected, because saved in an external file
```

Comment:
*can be/optionally

Also do not imply that this is really such a good protection. In the end it has to be readable by the backup user. It is just a small thing. And if admins configure it wrongly, it can still be bad, so…